docs/features.rst

```diff
@@ -225,3 +225,18 @@ Features related to the type system
      - Not supported
      - However, an ``Input`` can receive data published by other *Connext DDS* applications that use FlatData.
 
+Loading additional Connext DDS Libraries
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*Connector* supports features that require the loading of additional *Connext DDS*
+libraries, such as
+`Monitoring <https://community.rti.com/static/documentation/connext-dds/6.0.0/doc/manuals/connext_dds/html_files/RTI_ConnextDDS_CoreLibraries_UsersManual/index.htm#UsersManual/PartMonitoringLib.htm>`__
+and `Security Plugins <https://community.rti.com/static/documentation/connext-dds/current/doc/manuals/connext_dds/dds_security/html_files/RTI_SecurityPlugins_GettingStarted/index.htm>`__.
+
+In order for the Connector libraries to correctly load these libraries, it is necessary
+to either:
```

Comment:
Let's make it a bit more explicit:

To use RTI Connext DDS Add-ons you need an RTI Connext DDS installation. To configure your environment so that Connector can load these additional libraries. You can set your environment using:

```bash
> source <Connext DDS installation directory>/resource/scripts/rtisetenv_<architecture>.bash
```

or

```bash
> <Connext DDS installation directory>\resource\scripts\rtisetenv_<architecture>.bat
```

Or you can set your system's library path to `<Connext DDS installation directory>\lib\<architecture>\`

Reply:
(The option of copying the libs is too vague. We don't say which libs or where to copy them. So let's skip that)

Reply:
I would write/format this as follows:

To use RTI Connext DDS add-ons, you need an RTI Connext DDS installation. To configure your environment so that Connector can load these additional libraries: 

- Set your environment using:

  `  > source <Connext DDS installation directory>/resource/scripts/rtisetenv_<architecture>.bash`
  
  or

  `  > <Connext DDS installation directory>\resource\scripts\rtisetenv_<architecture>.bat`

- Or set your system's library path to `<Connext DDS installation directory>\lib\<architecture>\`